SDKTest/NeuroIDClassTests.swift

```diff
@@ -1023,7 +1022,7 @@ class NIDClientSiteIdTests: XCTestCase {
 
 class NIDSendTests: XCTestCase {
     func test_getCollectionEndpointURL() {
-        let expectedValue = "https://receiver.neuroid.cloud/c"
```

Comment:
Looks like this test failed its assertion. The init is not setting the dev endpoint prior to running the test?